### ISSUE TITLE
Glow: Tailwind

### ISSUE THREAD

**Issue by drewkybrown** [drewkybrown/weather-watcher#2]
I really enjoyed working with Tailwind. I found the process to be more manageable and clean then working with regular CSS styling. 
I felt like Tailwind allows you to do a lot more with a lot less and when going back over trying to restyle or adjust something the browser tools/extensions made finding the current code a lot easier.

For example:

The Tailwind Styling on my default background:

className="flex flex-col w-full h-screen bg-cover bg-center overflow-auto transition-all duration-500 ease-in-out"

VS the same styling but if I had used CSS:

.example-class-name {
    display: flex;
    flex-direction: column;
    width: 100%;
    height: 100vh;
    background-size: cover;
    background-position: center;
    overflow: auto;
    transition: all 0.5s ease-in-out;
}

**Comment by cfurlong15** [comment]
I agree that Tailwind has been a great addition to our workflow, and what you selected is a great example of implementing it. I'm impressed by the way you used it in a comprehensive way. As someone just viewing your code, I can look at it and have a pretty clear image of what you are trying to accomplish. 

Regarding the rest of your app, I really appreciate the way you established your conditions and used conditionals to render your data, especially in App.jsx. You managed to take a lot of possible outcomes and structure it in an organized and effective way. I'm taking notes for my own app.
